kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,8 +118,69 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
-## Uninstall the Voyager Operator
-After removing all the Voyager Ingress resources, uninstall the Voyager operator:
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at the ingress controller to access the WebLogic Server Administration Console through the SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on the WebLogic domain level
+
+If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to the model YAML file.
+```
+resources:
+     WebAppContainer:
+         WeblogicPluginEnabled: true
+```
+If you are using a WLST script to configure the domain, then the following modifications are needed to the respective PY script.
+```
+# Configure the Administration Server
+cd('/Servers/AdminServer')
+set('WeblogicPluginEnabled',true)
+...
+cd('/Clusters/%s' % cluster_name)
+set('WeblogicPluginEnabled',true)
+```
+### 2. Update the frontendRules section ingress resource
+Replace the string `weblogic-domain` with namespace of the WebLogic domain, the string `domain1` with domain UID and the string `adminserver` with name of the Administration Server in the WebLogic domain.
+
+**NOTE**: If you also have HTTP coming into ingress you will want to make sure that you remove any incoming WL-Proxy-SSL header. This protects you from a malicious user sending in a request and tricking WebLogic into thinking it's secure when it wasn't. Add the following rules in the Voyager ingress configuration to block any WL-Proxy headers coming from the client. In the following example the ingress resource will eliminate client header `WL-Proxy-Client-IP`.
```

Comment:
Please re-write this NOTE as follows:
**NOTE**: If you also have HTTP requests coming into an ingress, make sure that you remove any incoming `WL-Proxy-SSL` header. This protects you from a malicious user sending in a request to appear to WebLogic as secure when it isn't. Add the following rules in the Voyager ingress configuration to block `WL-Proxy` headers coming from the client. In the following example, the ingress resource will eliminate the client header `WL-Proxy-Client-IP`.